lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -1928,7 +1930,7 @@ private function getTypes($field, $value, ClassMetadata $class)
      *
      * @param mixed $value
      *
-     * @return array{mixed}
+     * @return mixed[]
```

Comment:
This is actually less precise because `array{mixed}` implies only one result in the array. You should keep `array{mixed}` in a @psalm-return